### PR TITLE
Google Code/Source: add boringssl-review.googlesource.com

### PR DESCRIPTION
**Pull request by lrinQVQ**

### CODE REVIEW

File: hosts [hunk 1]
```diff
@@ -9,7 +9,7 @@
 ## +-------------------------  >>d(' _ ')b<<  -------------------------+
 ## |                                                                   |
 ## |         Project: https://github.com/lennylxx/ipv6-hosts           |
-## |         Update : Tue, 31 Oct 2017 13:55:46 +0800                  |
+## |         Update : 五, 03 11月 2017 03:22:23 +0800                  |
```
Comment (lennylxx) [inline hosts]: This is not aligned to the previous version.

**Comment by lrinQVQ** [comment]
0.0 Please wait

File: hosts (hunk 1 above)
Comment (lrinQVQ) [reply hosts]: This is data common bug by locale language set to chinese
Maybe we need forced data format?such as:LC_ALL="C" date

File: hosts [hunk 2]
```diff
@@ -9,7 +9,7 @@
 ## +-------------------------  >>d(' _ ')b<<  -------------------------+
 ## |                                                                   |
 ## |         Project: https://github.com/lennylxx/ipv6-hosts           |
-## |         Update : Tue, 31 Oct 2017 13:55:46 +0800                  |
+## |         Update : Fri, 03 Nov 2017 19:07:43 +0800       |
```
Comment (lennylxx) [inline hosts]: Seems like you are missing some spaces this line.

Comment (lrinQVQ) [reply hosts]: Change again2333

**Pull request merged**